lib/array.js

```diff
@@ -86,18 +90,22 @@ const filter = (
 
 const EMPTY_ARR = 'Metasync: reduce of empty array with no initial value';
 
-const reduce = (
-  // Asynchronous reduce
-  items, // array, incoming
-  fn, // function, to be executed for each value in array
-  //   previous - value previously returned in the last iteration
-  //   current - current element being processed in the array
-  //   callback - callback for returning value back to function reduce
-  //   counter - index of the current element being processed in array
-  //   items - the array reduce was called upon
-  done, // function (optional), on done callback function(err, result)
-  initial // optional value to be used as first arpument in first iteration
-) => {
+// Asynchronous reduce
+//   items - <Array>, incoming
+//   fn - <Function>, to be executed for each value in array
+//     previous - <any>, value previously returned in the last iteration
+//     current - <any>, current element being processed in the array
+//     callback - <Function>, callback for returning value
+//         back to reduce function
+//     counter - <number>, index of the current element
+//         being processed in array
+//     items - <Array>, the array reduce was called upon
+//   done - <Function>, on done, optional
+//     err - <Error>
+//     result - <Array>
+//   initial - <number>, optional value to be used as first
```

Comment:
This is not specifically a number, this may be any value:
```suggestion
//   initial - <any>, optional value to be used as first
```